tests/componenttests/client/mocks/WebAudioPlayerModuleMock.h

```diff
@@ -101,6 +105,13 @@ class WebAudioPlayerModuleMock : public ::firebolt::rialto::WebAudioPlayerModule
         return response;
     }
 
+    ::firebolt::rialto::WebAudioGetBufferDelayResponse webAudioGetBufferDelayResponse(const uint32_t &delayFrames)
```

Comment:
I'd recommend passing int types by value rather than by reference (the final compiled code will probably be smaller and simpler)